collection-scripts/gather_nodes

```diff
@@ -17,20 +32,18 @@ oc create -f $CRD_MANIFEST
 oc adm policy add-scc-to-user privileged -n node-gather -z node-gather
 oc create -f $DAEMONSET_MANIFEST
 
-while true
-do
-    line=$(oc get ds node-gather-daemonset -o=custom-columns=DESIRED:.status.desiredNumberScheduled,READY:.status.numberReady --no-headers -n node-gather)
+NEXT_WAIT_TIME=0
+until check_node_gather_pods_ready || [ $NEXT_WAIT_TIME -eq 300 ]; do
+   sleep $(( NEXT_WAIT_TIME++ ))
```

Comment:
Not "sleep 1"?
With "sleep $(( NEXT_WAIT_TIME++ ))" we would wait for an arithmetic sequence from 1 to 300?

Reply:
you're right - fixed accordingly.